_maps/configs/nanotrasen_ranger.json

```diff
@@ -1,24 +1,23 @@
 {
 	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"prefix": "VISV",
 	"map_name": "Ranger-class Aid Vessel",
 	"faction": "/datum/faction/nt",
```

Comment:
`"faction": "/datum/faction/nt/vigilitas",`

Reply:
I should probably have this tested for..